services/ui-src/src/components/reports/ModalOverlayReportPage.test.tsx

```diff
@@ -107,7 +107,10 @@ describe("Test ModalOverlayReportPage (empty state)", () => {
       ...mockStateUserStore,
       ...mockMlrEntityStartedStore,
     });
-    render(modalOverlayReportPageInitialComponent);
+
+    await act(async () => {
```

Comment:
Fixes `Warning: An update to ReportProvider inside a test was not wrapped in act(...).`.